content/first-talk-in-spanish.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: My first talk in Spanish
+---
+# My first talk in Spanish
+This week I'll be giving a talk at NodeConf Argentina. It's going to be my first
+time giving a talk in Spanish. That's a huge deal for me, and it's all thanks to
+my friend Raul, thanks to whom I've come to see this as a responsibility I have
+to the latin american development community
```

Comment:
This sentence flows kind of weird. I wonder if you could split it up. Maybe something like the following? (ignore/edit as desired)
>That's a huge deal for me, and it's all thanks to my friend Raul. Thanks to him, I've come to see giving a tech talk in Spanish as a responsibility I have to the Latin American development community.

(I also capitalized "latin american" and added a period at the end.)